.Lib9c.Tests/JsonStatesLoaderTest.cs

```diff
@@ -1,23 +1,40 @@
 namespace Lib9c.Tests
 {
     using System;
+    using System.Collections.Generic;
     using System.IO;
+    using System.IO.Abstractions;
+    using System.IO.Abstractions.TestingHelpers;
+    using System.Text;
     using Bencodex;
     using Bencodex.Types;
     using Libplanet;
+    using Org.BouncyCastle.Utilities.Encoders;
     using Xunit;
-    using Zio;
-    using Zio.FileSystems;
 
     public class JsonStatesLoaderTest
     {
         [Fact]
         public void Load()
         {
-            IFileSystem fileSystem = new MemoryFileSystem();
             var codec = new Codec();
             Text barValue = (Text)"bar";
-            fileSystem.WriteAllText("/foo", $"{{\"foo\": \"{ByteUtil.Hex(codec.Encode(barValue))}\"}}");
+
+            byte[] Base64Encode(byte[] data)
+            {
+                var encoder = new Base64Encoder();
```

Comment:
How about using [`Convert.ToBase64String()`](https://docs.microsoft.com/ko-kr/dotnet/api/system.convert.tobase64string?view=netcore-3.1#System_Convert_ToBase64String_System_Byte___) instead of BouncyCastle's?

Reply:
I amended it in 3e75a3b .